src/components/10-atoms/spinner/index.js

```diff
@@ -0,0 +1,53 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import styles from './index.scss';
+
+class AXASpinner extends LitElement {
+  static get tagName() {
+    return 'axa-spinner';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      size: { type: Boolean, defaultValue: false },
```

Comment:
Like I wrote already: Size = false... what does that mean? No Size? So rename it to `small`.